feature/manager.go

```diff
@@ -179,6 +182,9 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 			raw.Unset(lnwire.SimpleTaprootChannelsOptionalStaging)
 			raw.Unset(lnwire.SimpleTaprootChannelsRequiredStaging)
 		}
+		if cfg.NoPeerStorage {
+			raw.Unset(lnwire.ProvideStorageOptional)
```

Comment:
Also unset the `ProvideStorageRequired`

Reply:
I did not add that in the default config yet. Just thought we should only add the optional flag for now.

Reply:
Look at *all* the code immediately above here though... when the config variable that disables PeerStorage is set, it should Unset both feature bits. Unset is an idempotent operation and so can be performed even when the bit is not already set.